queries/fortran/highlights.scm

```diff
@@ -5,147 +5,149 @@
 (comment) @comment
 
 [
-  (intrinsic_type)
-  "pointer"
-] @type
+ (intrinsic_type)
+ "allocatable"
+ "attributes"
+ "device"
+ "dimension"
+ "endtype"
+ "global"
+ "grid_global"
+ "host"
+ "import"
```

Comment:
I don't think all of these are `@type`

Reply:
`import` goes in `@keyword`, I think. The rest are subroutine / function attributes [[1]](https://docs.nvidia.com/hpc-sdk/compilers/cuda-fortran-prog-guide/#cfpg-sub-func-qualifiers). For example
```fortran
attributes(host) function foo(x) result(y)
    real, intent(in) :: x
    real :: y
    y = x + 1
end function foo
```
 Should I move them to `@attribute` or `@keyword`?

Reply:
Isn't `import` `@include`?

Reply:
moved `import` and `include` to `@include`. `global` etc to `@keyword`.